### DISCUSSION TITLE
Docker Compose Setup

### DISCUSSION THREAD

**Discussion by accforgithubtest** [evidence-dev/evidence#2385] [General]
## Request Type

- [ ] Walkthrough Request (new tutorial for a particular feature or use case)

## Description

This is a request to provide clear documentation of how to set up and use Evidence in a self-hosted (docker based) set up, without needing to go in to a development setup. Self hosting users will want to setup Evidence locally and Use the application (without having to go for a development set up).

I have seen the two other issues that have been raised and closed which provided the initial docker development setup, however there is still more needed for a easier user experience since they seem to be focused more on development set up than end user experience.

At a minimum, I think there should be -
1. A official docker image available at dockerhub (may be other publicly accessible places).
2. Documentation showing how to configure the various settings needed for evidence to run properly (volume/folder mappings, ports, environment variables, etc)
3. Configuring database connections and how to use multiple databases, etc.
4. Bonus details for reverse proxies set up, other advanced options, etc

Citing an example that I found easy to follow and use - https://vikunja.io/docs/full-docker-example/.

Something like this would greatly improve the user experience !

## Are you interested in contributing this improvement to the docs?
I am willing to update the documentation / provide written examples.
(I am new to Evidence, and also not a developer. However I am familiar with Docker (beginner level))

**Comment by jangorecki** [comment]
Considering how @archiewood described discussions vs issues, when opening discussions, this one seems to perfectly fit as an issue "Documentation Requests".
